.github/workflows/update-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Update tests list from master branch
+
+on:
+  push:
+    paths:
+      - 'tests/**'
+    branches:
+      - master
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: build and push Docker image
+        uses: ./.github/actions/build-and-push-docker
+        with:
+          registry: tip-tip-wlan-cloud-docker-repo.jfrog.io
+          registry_user: ${{ secrets.DOCKER_USER_NAME }}
+          registry_password: ${{ secrets.DOCKER_USER_PASSWORD }}
+
+  testsgen:
+    needs: [build]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Generate tests as allure results
+        run: docker run -i -t -v $PWD/allure-results:/allure-results tip-tip-wlan-cloud-docker-repo.jfrog.io/cloud-sdk-nightly:${{ github.run_id }} -c "cd tests; pytest -s -vvv --alluredir=/allure-results --skip-all"
+
+      - name: Generate Allure report
+        uses: ./.github/actions/generate-allure-report
+        with:
+          results_path: ./allure-results
+
+      - name: Publish Allure report to GitHub Pages
+        uses: ad-m/github-push-action@v0.6.0
```

Comment:
Same here, but unfortunately the maintainer didn't release a new version with the change yet (last release was over 2 years ago), so I switched to the `master` version. Don't know why we use an action for a simple git push in the first place, but that goes beyond the scope of this PR.